### PR TITLE
Ratings component

### PR DESCRIPTION
**Pull request by CSellers720**
#### Ratings component
- Ratings now displays stars.
- Rating component has been styled.
- NOTE: I'll need to link my rating count to the reviews component on the page.

### CODE REVIEW

File: client/src/components/ProductOptions/ProductOptions.jsx [hunk 1]
```diff
@@ -51,11 +52,29 @@ const DropHint = styled.div`
 `;
 
 const ProductOptions = (props) => {
+  const [productData, setProductData] = useState({
```
Comment (ccali11) [inline client/src/components/ProductOptions/ProductOptions.jsx:55]: Ah! React Hooks! Very good!

("Ah" in this case above is pronounced 'ah' as in when the doctor says open up and say 'ah' or as in "Ah! It all makes sense!" and not like the frightened utterance or interjection.)

File: client/src/components/ProductOptions/ProductOptions.jsx [hunk 2]
```diff
@@ -51,11 +52,29 @@ const DropHint = styled.div`
...
+    name: null,
+    rating: 0,
+    ratingcount: 0
+  });
+
+  useEffect(() => {
+    axios.get(`http://localhost:3030/static/${props.state.pid}`)
+      .then((results) => {
+        setProductData({
+          name: results.data.name
```
Comment (ccali11) [inline client/src/components/ProductOptions/ProductOptions.jsx]: Nice - so you're adjusting the state via the product name, yes? Looks good.

**Review by ccali11 — COMMENTED** [review]
Thanks for the React Hooks example! 👍

File: client/src/components/ProductOptions/ProductOptions.jsx (hunk 1 above)
Comment (CSellers720) [reply client/src/components/ProductOptions/ProductOptions.jsx:55]: ### AAAAAAAH! REACT HOOKS!!!
woah-dude

File: client/src/components/ProductOptions/ProductOptions.jsx (hunk 2 above)
Comment (CSellers720) [reply client/src/components/ProductOptions/ProductOptions.jsx]: Yup!

**Comment by CSellers720** [comment]
Merged in CLI

**Comment by CSellers720** [comment]
Merged in CLI.

**Pull request closed without merging**